solr/core/src/test-files/solr/collection1/conf/solrconfig-cache-enable-disable.xml

```diff
@@ -70,6 +70,10 @@
 
     <queryResultWindowSize>10</queryResultWindowSize>
 
+    <useCircuitBreakers>false</useCircuitBreakers>
+
+    <memoryCircuitBreakerThreshold>100</memoryCircuitBreakerThreshold>
```

Comment:
Does this have to be set when useCircuitBreakers is false? I would like our configs to be more tolerant.

Reply:
I dont think so, since it is checked when circuit breakers are enabled. I put this config here for completion.